### ISSUE TITLE
Fridge opening backwards on c8m1_apartments

### ISSUE THREAD

**Issue by actin0** [Tsuey/L4D2-Community-Update#420]
### Description

Similar to #418 . If a player opens a fridge while aiming at the 'back' of the func_door_rotating (fridge door), it will open backwards. 'One Way' flag should fix it. Interesting thing: on a fridge in the beginning of the map, only freezer door doesn't have One Way flag, the main fridge door opens fine from any angle. Literally unplayable.
![clu1](https://github.com/Tsuey/L4D2-Community-Update/assets/128401041/deb84bfe-e573-4dff-972a-3a4b2983b5da)
![clu3](https://github.com/Tsuey/L4D2-Community-Update/assets/128401041/14cb8739-cfef-4dc4-966d-59513763c7ad)
Second fridge (with only freezer door attached), frezeer door doesn't have One Way flag, but you can't open it backwards because of the surrounding furniture, unless player uses noclip. 
![clu2](https://github.com/Tsuey/L4D2-Community-Update/assets/128401041/bff96baf-046b-459b-831a-4cef8176e121)

### Reproduction steps

To open first fridge's freezer door backwards, just climb on the fridge and position your crosshair on the 'back' of the door. The second fridge's freezer door can't open backwards without using noclip.

### Additional files

https://github.com/Tsuey/L4D2-Community-Update/assets/128401041/26aa6069-a0b4-42d6-8dd7-60e1a0fab567

**Comment by Tsuey** [comment]
We will fix every last one of these pesky fridges.

**Comment by alexiscoutinho** [comment]
Continuation of #418 and imo duplicate.

@actin0 I highly recommend that you merge these issues as they cover the same problem.

**Comment by actin0** [comment]
Closed, because duplicate of #418 imo

**Comment by alexiscoutinho** [comment]
> Closed, because duplicate of #418

I still suggest that you include this video and update the description and title of the original issue to properly account for this map. ;)

**Comment by actin0** [comment]
Just saw your comment. Merged #420 (this issue) completely with #418
